frontend/src/HomePage/SignHandlers/SignUp.css

```diff
@@ -0,0 +1,60 @@
+.topbar {
```

Comment:
nit: could we use camelCase for these class names? camelCase is pretty widely used and helps improve readability.

Reply:
As mentioned in your other comment, I think using hyphens for CSS seems to be the common from my observations so I'd go with hyphens (but use camel case in other code)